src/pages/countryDetail/countryDetail.tsx

```diff
@@ -1,19 +1,19 @@
 import { t } from 'i18next';
 import React from 'react';
 import { AvailableISO, FlagIcon, ISO2RFC } from '../../others/components/FlagIcon';
-import { ServiceList } from '../../others/components/ServiceList';
+import { ServiceList } from './ServiceList';
 import { Spacer } from '../../others/components/Spacer';
-import { Country } from '../../others/contexts/api';
-import styles from "./countryDetail.module.css";
-
+import { Country } from '../../others/contexts/country';
+import styles from './countryDetail.module.css';
+import { SERVICE_TYPE } from '../../others/contexts/service';
 
 export interface CountryDetailProps {
   country: Country;
 }
 
 export function CountryDetail({ country }: CountryDetailProps) {
-  const inhouse = country?.info?.services?.inhouse;
-  const external = country?.info?.services?.external;
+  const inhouse = country.services.filter((service) => service.type !== SERVICE_TYPE.EXTERNAL);
+  const external = country.services.filter((service) => service.type === SERVICE_TYPE.EXTERNAL);
```

Comment:
This should filter for the service origin rather than its type

Reply:
What is service origin? Don't see a column that specifies that.

Reply:
This would be the inhouse/external distinction I refereed to earlier

Reply:
👍 I responded to this comment prior to reading the one above. :)